advanced/macros.rst

```diff
@@ -69,3 +69,7 @@ To apply a macro to many tickets at the same time:
    
    When running it from the overviews page, Zammad will always stay on the
    overviews page.
+
+   ⚠️ **Macro limit:** In the overview page, the macros that shows up have limit
+   depending on the screen size, however, as much as 20 macros could show no
+   matter the screen size.
```

Comment:
Hi @MrGeneration Please let me know if this wording is okay. It is related to https://github.com/zammad/zammad/issues/3359 and also `MR !1919` on gitlab

Reply:
Is that a hard coded limit?  I find it difficult to put such a remark into the documentation. It may be confusing. 
Are there break points?

Reply:
How would you suggest we put it? 

Also, it's not hard coded, it depends on the screen size. The MR tried to fix in as much as possible, from what I tested it could take as much as 24 macros currently.

Reply:
The only break point is to make the macro circles smaller on smaller screen sizes. This also help fit in more macros.

Reply:
Thanks for your patience.
I believe that the macro limit remark is too technical for the *user documentation*.

---

If I understand correctly you want to tell end user the following:

```
.. note:: **🎚 Variable macro display**

   Depending on your screen size, Zammad shows up to 20 Macros for selection.
```

Correct?
This would probably a rather technical note that is relevant for administrators. 
What do @martini and @thorsteneckel think?

Reply:
I think you will come up with the best assessment on what to write where 👍

Reply:
What @thorsteneckel said 😊